src/api/app/views/webui2/webui/project/_side_links.html.haml

```diff
@@ -0,0 +1,50 @@
+%ul.list-unstyled
+  - if is_maintenance_project
+    = render partial: 'webui2/webui/project/side_links/maintenance_project', locals: { open_maintenance_incidents: open_maintenance_incidents,
+                                                                                       project: project,
+                                                                                       maintained_projects: maintained_projects }
+  - else
+    - if nr_of_problem_packages && nr_of_problem_packages > 0
```

Comment:
`nr_of_problem_packages.try(:positive?)` :bowtie:

Reply:
I didn't know about `positive?`, but I'm not sure... I think it's more readable now (even though it's longer)

Reply:
I prefer to keep it as it is.

Reply:
This is introducing new Rubocop offenses: https://rubocop.readthedocs.io/en/latest/cops_style Rubocop doesn't complain because it is set to fails in the **todo** because we have 41 offenses, but it is enabled. if you don't like `try` is ok, but please fix the `positive?`

Reply:
Damn you Rubocop!!! (Fine, I fixed it)